src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dayRent = 40;
+  let totalRental = 0;
+
+  if (days >= 7) {
+    totalRental = dayRent * days - 50;
+  } else if (days >= 3) {
+    totalRental = dayRent * days - 20;
```

Comment:
Again, avoid using literals in expressions. Use constants to define the discount values and the thresholds for the number of days, according to the checklist item 2.